assets/js/util/tracking/createEnableTracking.js

```diff
@@ -36,6 +41,21 @@ export default function createEnableTracking( config, dataLayerTarget ) {
 		document.head.appendChild( scriptTag );
 
 		dataLayerPush( 'js', new Date() );
-		dataLayerPush( 'config', config.trackingID );
+		dataLayerPush( 'config', config.trackingID, {
+			send_page_view: config.isSiteKitScreen,
+		} );
+		hasInsertedTag = true;
+	};
+
+	/**
+	 * Enables tracking.
+	 */
+	function enableTracking() {
+		initializeSnippet();
+	}
```

Comment:
@aaemnnosttv this function seems to be useless after adding the `initializeSnippet` one. Do we need to remove it? If we remove there is no sense to return an object anymore, right? Then why not just update the existing returning function to protect from multiple injections and leave it as is?

Reply:
This function should still set `config.trackingEnabled = true` as it did before (as mentioned above) but I think we can move this up to be like `disableTracking` instead (in `createTracking`), and this module can be specific to only creating `initializeSnippet` instead.

Reply:
https://github.com/google/site-kit-wp/pull/4142/commits/4faffa2ddd8f92d4cf9aeae34ec7c9d2c948ce3f